modules/openapi-generator/src/main/resources/r/modelGeneric.mustache

```diff
@@ -330,7 +330,7 @@
           {{/isArray}}
           {{#isMap}}
           {{#isPrimitiveType}}
-            {{#isNumeric}}%d{{/isNumeric}}{{^isNumeric}}{{^isBoolean}}"{{/isBoolean}}%s{{^isBoolean}}"{{/isBoolean}}{{/isNumeric}}
+            {{#isNumeric}}%d{{/isNumeric}}{{^isNumeric}}{{^isBoolean}}{{/isBoolean}}%s{{^isBoolean}}{{/isBoolean}}{{/isNumeric}}
```

Comment:
thanks for the PR. for this change, can we add a test model in the test spec to ensure it's tested moving forward?

Reply:
Please clarify what you need, or link a PR that did something similar. I would be glad to contribute some testing!

Reply:
please update modules/openapi-generator/src/test/resources/3_0/r/petstore.yaml with test schemas (models) to ensure the above change works as expected (and then regenerate the samples)

Reply:
I added a minimal map schema that leads to the generation of the `toJSONString` function I mentioned above. Please see if this is what you wanted.